src/main/augurUIServer.js

```diff
@@ -126,6 +126,7 @@ AugurUIServer.prototype.onStopUiServer = function () {
 
 AugurUIServer.prototype.restart = function (event) {
   if (this.server !== null) this.onStopUiServer()
+  if (this.httpListener) this.httpListener.close()
```

Comment:
https://nodejs.org/api/net.html#net_server_close_callback

Should we restart the server inside a callback?

Reply:
didn't know there was a callback. that makes sense, just in case something gets hung up.